seatunnel-common/src/main/java/org/apache/seatunnel/common/Handover.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.seatunnel.connectors.seatunnel.pulsar.source.reader;
+package org.apache.seatunnel.common;
```

Comment:
Is it possible to use it in non-connector modules?
I mean it might be better to move to `connector-common` module